cmd/common/registry.go

```diff
@@ -420,6 +422,10 @@ func buildCommandFromGadget(
 						}
 					}
 				}
+				// hide columns by tag (e.g. kubernetes, runtime) if requested by the caller
+				if len(hiddenColumnTags) > 0 {
+					hiddenTags = append(hiddenTags, hiddenColumnTags...)
+				}
 				requestedColumns = parser.GetDefaultColumns(hiddenTags...)
```

Comment:
We should pass these `hiddenTags` also [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/0d5ed51e1f9e0d67d37b1d3e6469ebd328838e44/cmd/common/registry.go#L148). Otherwise, we are showing the incorrect default columns in the `help` command:

```bash
$ sudo ./ig trace open -h
Trace open system calls
...
  -o, --output string          Output format (columns, json, jsonpretty, yaml).

                               Columns (columns)
                                 The output of the gadget is formatted in human readable columns.
                                 You can optionally specify the columns to output using '-o columns=col1,col2,col3' etc.
                                   Available columns:
                                     comm
                                     err
                                     fd
                                     flags
                                     fullPath (requires --full-path)
                                     gid
                                     k8s.container
                                     k8s.hostnetwork
                                     k8s.namespace
                                     k8s.node
                                     k8s.pod
                                     mntns
                                     mode
                                     path
                                     pid
                                     ret
                                     runtime.containerId
                                     runtime.containerImageName
                                     runtime.containerName
                                     runtime.runtimeName
                                     timestamp
                                     uid
                                   Default columns: runtime.containerName,k8s.node,k8s.namespace,k8s.pod,k8s.container,pid,comm,fd,err,path,fullPath
```